src/components/table.js

```diff
@@ -457,31 +554,35 @@ export const HeaderCell = props => {
 }
 
 export const Sortable = ({ sort, field, onSort, children }) => {
-  return div({
-    style: { flex: 1, display: 'flex', alignItems: 'center', cursor: 'pointer', width: '100%' },
-    onClick: () => onSort(Utils.nextSort(sort, field))
+  return h(IdContainer, [id => h(Clickable, {
```

Comment:
where does `id` come from?

Reply:
The `IdContainer` generates stateful unique ids and passes them down to its children. I just copied and pasted an existing design pattern I found in use throughout the site.

I'm not entirely sure how that `id` gets passed in there, and what would happen if I had more than one child in that array; would they each be called with the `id` parameter or would only the first one be? But it seems to do what I want when written like this.